manifests/apps/rhoai/rhoai-app.yaml

```diff
@@ -0,0 +1,20 @@
+  getStartedLink: 'https://access.redhat.com/documentation/en-us/red_hat_openshift_ai_cloud_service/1/html/openshift_ai_tutorial_-_fraud_detection_example/index'
+  displayName: RHOAI
+  support: red hat
+  provider: Red Hat
+  hidden: true
+  docsLink: 'https://access.redhat.com/documentation/en-us/red_hat_openshift_ai_cloud_service/1/html/openshift_ai_tutorial_-_fraud_detection_example/index'
```

Comment:
Leave `docksLink`, `getStartedMarkDown` and `description` empty (as an empty string) to avoid render a card in the explore card.